UnityProject/Assets/Scripts/Construction/ConveyorBelts/ConveyorBelt.cs

```diff
@@ -309,4 +310,15 @@ public void ServerPerformInteraction(HandApply interaction)
 				});
 		}
 	}
+
+	//changes the rendered sprite in editor based on the value set in Directional
+#if UNITY_EDITOR
+	private void Update()
+	{
```

Comment:
You can probably change it to `OnValidate`
https://docs.unity3d.com/ScriptReference/MonoBehaviour.OnValidate.html

Reply:
Thanks, I'll try it out.